node/src/reactor.rs

```diff
@@ -283,6 +283,11 @@ pub(crate) trait ReactorEvent: Send + Debug + From<ControlAnnouncement> + 'stati
     /// [`ControlAnnouncement`](`crate::effect::announcements::ControlAnnouncement`) if the event
     /// is indeed a control announcement variant.
     fn as_control(&self) -> Option<&ControlAnnouncement>;
+
+    /// Returns a cheap but human-readable description of the event.
+    fn description(&self) -> &'static str {
+        "anonymous event"
```

Comment:
```suggestion
        "unknown event type"
```
?

Reply:
It's not exactly unknown though? :)

Reply:
I wanted something that does not make the user give up immediately, but rather realize that there might be something that one can do about it!